### PR TITLE
Company number localisation

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

Addressing #1677. Updated some of the localisation of "ABN" into "Company Number" for the UK.

Update: Also changed "ACN" to "Charity Number".

#### What should we test?

See screenshots in original issue. Testing with en-GB locale.

### CODE REVIEW

File: config/locales/en-GB.yml [hunk 1]
```diff
@@ -212,6 +212,11 @@ en-GB:
         status: Status
         manage: Manage
       form:
+        business_details:
+          abn: "Company Number"
+          abn_placeholder: "eg. 99123456789"
+          acn: "ACN"
```
Comment (mkllnk) [inline config/locales/en-GB.yml]: You still have ACN here?

**Comment by daniellemoorhead** [comment]
@enricostano given @mkllnk's response are you  happy to change your request and approve the review? And @Matt-Yorkley if you could get the remaining ACN reference @mkllnk found sorted then we could approve the change and get Oliver to test it in time for the July release next Friday.

**Comment by daniellemoorhead** [comment]
Hey @Matt-Yorkley reckon you could stage this for @OliverUK to test? If it passes then @mkllnk can merge to master it next week.

File: config/locales/en-GB.yml (hunk 1 above)
Comment (Matt-Yorkley) [inline config/locales/en-GB.yml]: It makes sense for ABN to become Company Number for the UK, but I don't think we have an alternate equivalent for ACN...

Comment (Matt-Yorkley) [inline config/locales/en-GB.yml]: Looks like Nick suggested Charity Number, I'll go with that.

**Comment by OliverUK** [comment]
OK so if this is simply a translation issue, I've checked on staging and it displays what it should. I've also entered Stroudco's company number and saved it. 
screen shot 2017-07-24 at 15 11 23

In the original issues this links in with invoices not sending. I don't usually use invoices in OFN but I have tried on staging to send an invoice for an order I did some time ago and got a slug error page, so can't comment on this further.

@lin-d-hop  you showed me how to do changes in translation files and we changed "on demand" to "unlimited" in the UK but his has never made it on to OFN UK.

**Pull request merged**